packager/react-native-xcode.sh

```diff
@@ -46,7 +46,14 @@ fi
 # npm global install path may be a non-standard location
 PATH="$(npm prefix -g)/bin:$PATH"
 
-react-native bundle \
+# check for NPM-installed react-native-cli
+if [[ -d ./node_modules/react-native-cli ]] && [ -z "$REACT_NATIVE_PATH" ]; then
```

Comment:
We rely on cwd inside Xcode because I don't see any other way. In `react-native-xcode.sh` we can actually resolve the path to `local-cli/cli.js` relative to `packager/react-native-xcode.sh` location.